app/webpacker/components/CompetitionResultSubmission/ImportResultsData/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { Accordion, Container, Message } from 'semantic-ui-react';
+import UploadResultsJson from '../UploadResultsJson';
+import WCAQueryClientProvider from '../../../lib/providers/WCAQueryClientProvider';
+
+export default function Wrapper({
+  competitionId,
+  alreadyHasSubmittedResult,
+  isWrtViewing,
+}) {
+  return (
+    <WCAQueryClientProvider>
+      <ImportResultsData
+        competitionId={competitionId}
+        alreadyHasSubmittedResult={alreadyHasSubmittedResult}
+        isWrtViewing={isWrtViewing}
+      />
+    </WCAQueryClientProvider>
+  );
+}
+
+function ImportResultsData({
+  competitionId,
+  alreadyHasSubmittedResult,
+  isWrtViewing,
+}) {
+  const [activeAccordion, setActiveAccordion] = useState(!alreadyHasSubmittedResult);
+
+  return (
+    <Container fluid>
+      <Accordion fluid styled>
+        <Accordion.Title
+          active={activeAccordion}
+          onClick={() => setActiveAccordion((prevValue) => !prevValue)}
+        >
+          Import Results Data
+        </Accordion.Title>
+        <Accordion.Content active={activeAccordion}>
+          {alreadyHasSubmittedResult
+            ? (
+              <Message warning>
+                Some results are already there, importing results data again will override all
```

Comment:
This message is a bit misleading. It is easily possible to have some results which are uploaded, but not yet posted. I think "some inbox results are already there" is the smallest possible fix, but may lead to confusion with ordinary Delegates who don't know anything about our inbox and posting processes. So what about "Some results have already been uploaded before"?